samples/suit/smp_transfer/README.rst

```diff
@@ -268,91 +277,205 @@ After programming the sample to your development kit and updating the sequence n
-               18.99 KiB / 244.57 KiB [============>-------------------------------------------------------------------------------------------------------------------------------------------------]   7.76% 11.83 KiB/s 00m19s
-               66.56 KiB / 244.57 KiB [==========================================>-------------------------------------------------------------------------------------------------------------------]  27.21% 18.44 KiB/s 00m09s
-               112.12 KiB / 244.57 KiB [=======================================================================>-------------------------------------------------------------------------------------]  45.84% 19.97 KiB/s 00m06s
-               154.08 KiB / 244.57 KiB [==================================================================================================>----------------------------------------------------------]  63.00% 20.22 KiB/s 00m04s
-               197.40 KiB / 244.57 KiB [==============================================================================================================================>------------------------------]  80.71% 20.51 KiB/s 00m02s
-               241.16 KiB / 244.57 KiB [=================================================================================================================================================================>--]  98.60% 20.74 KiB/s
-               Done
+            .. code-block:: console
+
+               [00:00:07] ###### 100% [/dev/ttyACM0] Uploaded
+
       #. If you have built the application with :ref:`external flash support <nrf54h_suit_sample_extflash>`, upload the cache partition to the external flash using the following command:
 
          .. code-block:: console
 
-            mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image upload -n 2 build/DFU/dfu_cache_partition_1.bin
+            nrfutil suit upload-cache-raw --serial-port=/dev/ttyACM0 --cache-file build/dfu_cache_partition_1.bin --pool 2
+
 
             .. note::
-               the ``-n 2`` parameter uploads to DFU cache partition 1 (where image 0 is the envelope and image 1 is the cache partition 0).
+               the ``--pool 2`` parameter uploads to DFU cache partition 1 (where image 0 is the envelope and image 1 is the cache partition 0).
 
       #. Start the installation of the new firmware as follows:
 
          .. code-block:: console
 
-            mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image confirm
+            nrfutil suit install --serial-port=/dev/ttyACM0
 
-      #. Read the version and digest of the uploaded root manifest with MCUmgr:
+      #. Read the sequence number of the uploaded root manifest with nrfutil:
 
          .. code-block:: console
 
-             mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image list
+             nrfutil suit manifests --serial-port=/dev/ttyACM0
```

Comment:
```suggestion
             nrfutil suit manifests --serial-port COM4
```

Reply:
Will change to COM7 like in nrfutil docs